docs/pages/versions/v38.0.0/sdk/notifications.md

```diff
@@ -218,7 +211,7 @@ async function registerForPushNotificationsAsync() {
 
 </SnackInline>
 
-> **Note** this demo might not fully work when you run it from the snack, because `app.json` needs to contain `useNextNotificationsApi` flag. Unfortunately, the snack doesn't support custom `app.json`.
+> **Note** this demo will not work with **remote** notifications (sent via the Expo Notifications service) on Android when you run it from the snack, because `app.json` needs to contain the `useNextNotificationsApi` flag. Unfortunately, snack doesn't support custom `app.json` files.
```

Comment:
```suggestion
> **Note** this demo will not work with **remote** notifications (sent via the Expo Notifications service) on Android when you run it from the snack, because `app.json` needs to contain the `useNextNotificationsApi` flag. Unfortunately, Snack doesn't support custom `app.json` files.
```